### DISCUSSION TITLE
3770K无法使用有线网卡

### DISCUSSION THREAD

**Discussion by xwenyan888** [community/community#52967] [Programming Help]
### Select Topic Area

Question

### Body

2012款iMac A1418升级了I7 3770K后，用opencore-legacy-patcher引导安装系统（不管哪个版本），会导致无法识别有线网卡。

**Comment by queenofcorgis** [comment]
Hi @xwenyan888 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
